builtin/providers/aws/resource_aws_s3_bucket.go

```diff
@@ -38,13 +38,22 @@ func resourceAwsS3Bucket() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"index_document": &schema.Schema{
 							Type:     schema.TypeString,
-							Required: true,
+							Optional: true,
 						},
 
 						"error_document": &schema.Schema{
 							Type:     schema.TypeString,
 							Optional: true,
 						},
+
+						"redirect_all_requests_to": &schema.Schema{
+							Type: schema.TypeString,
+							ConflictsWith: []string{
+								"website.0.index_document",
+								"website.0.error_document",
+							},
```

Comment:
The build is failing because `InternalValidate` in `helper/schema/schema.go` doesn't support this nested notation for `ConflictsWith`, even though in usage it seems to work correctly.